static/locales/ur.yaml

```
@@ -0,0 +1,790 @@
+Delete: 'حذف کریں۔'
+Select all: 'تمام منتخب کریں'
+Reload: 'دوبارہ لوڈ کریں۔'
+Force Reload: 'زبردستی دوبارہ لوڈ کریں۔'
+Toggle Developer Tools: 'ڈویلپر ٹولز کو ٹوگل کریں۔'
+Actual size: 'اصل سائز'
+Zoom in: 'زوم ان کریں۔'
+Zoom out: 'دور کرنا'
+Toggle fullscreen: 'پوری اسکرین کو ٹوگل کریں۔'
+Window: 'کھڑکی'
+Minimize: 'کم سے کم کرنا'
+Close: 'بند کریں'
+Back: 'پیچھے'
+Forward: 'آگے'
+Open New Window: 'نئی ونڈو کھولیں۔'
+
+Version $ is now available!  Click for more details: 'ورژن $ اب دستیاب ہے! کلک کریں۔
+   مزید تفصیلات کے لیے'
+Download From Site: 'سائٹ سے ڈاؤن لوڈ کریں۔'
+A new blog is now available, $. Click to view more: 'ایک نیا بلاگ اب دستیاب ہے، $.
+   مزید دیکھنے کے لیے کلک کریں۔'
+Are you sure you want to open this link?: 'کیا آپ واقعی اس لنک کو کھولنا چاہتے ہیں؟'
+
+# Search Bar
+Search / Go to URL: 'تلاش کریں / یو آر ایل پر جائیں۔'
+Search Bar:
+  Clear Input: 'ان پٹ صاف کریں۔'
+  # In Filter Button
+Search Filters:
+  Search Filters: 'فلٹرز تلاش کریں۔'
+  Sort By:
+    Sort By: ''
+    Most Relevant: ''
+    Rating: ''
+    Upload Date: ''
+    View Count: ''
+  Time:
+    Time: ''
+    Any Time: ''
+    Last Hour: ''
+    Today: ''
+    This Week: ''
+    This Month: ''
+    This Year: ''
+  Type:
+    Type: ''
+    All Types: ''
+    Videos: ''
+    Channels: ''
+    #& Playlists
+  Duration:
+    Duration: ''
+    All Durations: ''
+    Short (< 4 minutes): ''
+    Long (> 20 minutes): ''
+  # On Search Page
+  Search Results: ''
+  Fetching results. Please wait: ''
+  Fetch more results: ''
+  There are no more results for this search: 'اس تلاش کے مزید نتائج نہیں ہیں۔'
+# Sidebar
+Subscriptions:
+    # On Subscriptions Page
+  Subscriptions: ''
+  # channels that were likely deleted
+  Error Channels: ''
+  Latest Subscriptions: ''
+  This profile has a large number of subscriptions.  Forcing RSS to avoid rate limiting: 'یہ
+     پروفائل میں سبسکرپشنز کی ایک بڑی تعداد ہے۔ شرح کو محدود کرنے سے بچنے کے لیے آر ایس ایس کو مجبور کرنا'
+  'Your Subscription list is currently empty. Start adding subscriptions to see them here.': 'آپ کا
+     رکنیت کی فہرست فی الحال خالی ہے۔ انہیں یہاں دیکھنے کے لیے سبسکرپشنز شامل کرنا شروع کریں۔'
+  'Getting Subscriptions. Please wait.': 'سبسکرپشنز حاصل کرنا۔ برائے مہربانی انتظار کریں.' 
+  Refresh Subscriptions: 'سبسکرپشنز کو ریفریش کریں۔'
+  Load More Videos: 'مزید ویڈیوز لوڈ کریں۔'
+More: 'مزید'
+Channels:
+  Channels: 'چینلز'
+  Title: 'چینل کی فہرست'
+  Search bar placeholder: 'چینلز تلاش کریں۔'
+  Count: '$چینل ملا۔'
+  Empty: 'آپ کے چینل کی فہرست فی الحال خالی ہے۔'
+  Unsubscribe: 'ان سبسکرائب کریں۔'
+  Unsubscribed: '$ کو آپ کی سبسکرپشنز سے ہٹا دیا گیا ہے۔'
+  Unsubscribe Prompt: 'کیا آپ واقعی "$" سے ان سبسکرائب کرنا چاہتے ہیں؟'
+Trending:
+  Trending: ''
+  Default: ''
+  Music: ''
+  Gaming: ''
+  Movies: ''
+  Trending Tabs: ''
+Most Popular: ''
+Playlists: ''
+User Playlists:
+  Your Playlists: ''
+  Playlist Message: 'یہ صفحہ مکمل طور پر کام کرنے والی پلے لسٹس کا عکاس نہیں ہے۔ یہ صرف
```

remove outer 's as string contains one in it
```suggestion
  Playlist Message: یہ صفحہ مکمل طور پر کام کرنے والی پلے لسٹس کا عکاس نہیں ہے۔ یہ صرف
```